proto/desmos/subspaces/v1beta1/msgs.proto

```diff
@@ -0,0 +1,160 @@
+}
+
+// MsgEditSubspaceResponse defines the Msg/EditSubspace response type
+message MsgEditSubspaceResponse{}
+
+// ___________________________________________________________________________________________________________________
+
+// MsgAddAdmin represents the message used to add an admin to a specific subspace
+message MsgAddAdmin {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string subspace_id = 1 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.jsontag) = "subspace_id",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+
+  string new_admin = 2 [(gogoproto.moretags) = "yaml:\"new_admin\""];
+  string owner = 3 [ (gogoproto.moretags) = "yaml:\"owner\"" ];
+}
+
+// MsgAddAdminResponse defines the Msg/AddAdmin response type
+message MsgAddAdminResponse {}
+
+// ___________________________________________________________________________________________________________________
+
+// MsgRemoveAdmin represents the message used to remove an admin from a specific subspace
+message MsgRemoveAdmin {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string subspace_id = 1 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.jsontag) = "subspace_id",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+
+  string admin = 2 [(gogoproto.moretags) = "yaml:\"admin\""];
+  string owner = 3 [(gogoproto.moretags) = "yaml:\"owner\""];
+}
+
+// MsgRemoveAdminResponse defines the Msg/RemoveAdmin response type
+message MsgRemoveAdminResponse{}
+
+// ___________________________________________________________________________________________________________________
+
+// MsgRegisterUser represents the message used to allow a user to post inside a specific subspace
+message MsgRegisterUser {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string user = 1 [ (gogoproto.moretags) = "yaml:\"user\"" ];
+  string subspace_id = 2 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.jsontag) = "subspace_id",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+  string admin = 3 [ (gogoproto.moretags) = "yaml:\"admin\"" ];
+}
+
+// MsgRegisterUserResponse defines the Msg/MsgRegisterUser response type
+message MsgRegisterUserResponse {}
+
+// ___________________________________________________________________________________________________________________
+
+// MsgBlockUser represents the message used to block a user to post inside a specific subspace
+message MsgBlockUser {
+  option (gogoproto.equal) = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string user = 1 [ (gogoproto.moretags) = "yaml:\"user\"" ];
+  string subspace_id = 2 [
+    (gogoproto.customname) = "SubspaceID",
+    (gogoproto.jsontag) = "subspace_id",
+    (gogoproto.moretags) = "yaml:\"subspace_id\""
+  ];
+  string admin = 3 [ (gogoproto.moretags) = "yaml:\"admin\"" ];
```

Comment:
I think this should better be `user` instead of `admin`

Reply:
same as above. Maybe we should name the `user` field `address` and `admin` -> `user`?

Reply:
@bragaz Didn't see we already have a `user` field. My bad